util/domd

```diff
@@ -32,9 +32,10 @@ else
     ${PERL} $TOP/util/clean-depend.pl < Makefile > Makefile.new
     RC=$?
 fi
-if ! cmp -s Makefile Makefile.new; then
+if ! cmp -s Makefile.save Makefile.new; then
```

Comment:
Right. Makefile is now modified, and we need to choose whether to overwrite it with the new one (which might be equivalent to `Makefile`, but not to `Makefile.save`), or revert to `Makefile.saved`.